### PR TITLE
more keystore tests

### PR DESCRIPTION
**Pull request by winsvega**
according to this page
https://github.com/ethereum/wiki/wiki/Web3-Secret-Storage-Definition

### CODE REVIEW

File: test/libdevcrypto/SecretStore.cpp [hunk 1]
```diff
@@ -215,5 +216,122 @@ BOOST_AUTO_TEST_CASE(recode)
...
+		BOOST_CHECK_EQUAL(uuid, h128("3198bc9c66725ab3d9954942343ae5b6"));
+		BOOST_CHECK(contentsString(importFile) == keyData);
+		BOOST_CHECK_EQUAL(priv, toHex(store.secret(uuid, [&](){ return password; }).makeInsecure()));
+		BOOST_CHECK_EQUAL(store.keys().size(), 1);
+		BOOST_CHECK_EQUAL(store.address(uuid), Address("008aeeda4d805471df9b2a5b0f38a0c3bcba786b"));
+	}
+	fs::remove(importFile);
+}
+
+BOOST_AUTO_TEST_CASE(vector_Scrypt2, *utf::expected_failures(2))
```
Comment (chfast) [inline test/libdevcrypto/SecretStore.cpp]: Why 2 failures here?

File: test/libdevcrypto/SecretStore.cpp [hunk 2]
```diff
@@ -215,5 +216,122 @@ BOOST_AUTO_TEST_CASE(recode)
 	}
 }
 
+BOOST_AUTO_TEST_CASE(vector_PBKDF2SHA256, *utf::expected_failures(1))
```
Comment (chfast) [inline test/libdevcrypto/SecretStore.cpp]: What this `vector_` prefix in the test name suppose to mean? Name it `KeyImport` or something like that.

File: test/libdevcrypto/SecretStore.cpp (hunk 1 above)
Comment (winsvega) [inline test/libdevcrypto/SecretStore.cpp]: old test format

Comment (chfast) [inline test/libdevcrypto/SecretStore.cpp]: I don't understand. Does it mean it does not import it at all?

**Pull request merged**